persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/CassandraPersistenceActivator.java

```diff
@@ -152,7 +151,10 @@ public void start(BundleContext context) {
     props.put("Identifier", "CassandraPersistence");
 
     try {
-      cassandraDB.initialize(makeConfig());
+      // Throw away data directory since stargate is ephemeral anyway
+      File baseDir = Files.createTempDirectory("stargate-cassandra").toFile();
```

Comment:
maybe it will be more clear if we would create `stargate-cassandra-3_11` directory (and `stargate-cassandra-4` for the second Cassandra backend?)

Reply:
Curious, why was the `baseDir` generation moved out of `makeConfig()`? It's not used again.

Reply:
Sure. I'll do that

Reply:
Not sure it really matters since only one persistence will be ran at a time and once it exits it'll clean up the directory.

Reply:
I make the change anyway, for the sake of clarify in case someone is browsing those temp dirs.